criu/util.c

```diff
@@ -1139,6 +1139,24 @@ int fd_has_data(int lfd)
 	return ret;
 }
 
+void fd_nonblocking(int fd, bool on)
```

Comment:
maybe fd_set_nonblocking?

Reply:
Thank you, fd_set_nonblocking is better.